README.md

```diff
@@ -1,2 +1,80 @@
-# Awesome-Books-App
-Html &amp; Js
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Awesome Book App</b></h3>
+
+</div>
+
+# 📖 Capstone 1 <a name="Awesome Book App"></a>
+
+> A book app that adds and deletes books from the list.
+
+## 🛠 Built With <a name="built-with">HTML, CSS, Sass & Javascript</a>
+
+- It has implemented Js Dynamic function to take data and display the books info.
+
+### Key Features <a name="key-features"></a>
+
+- Clean and interactive
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+All you need VScompiler or a compiler of your choice.
+
+### Clone
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/felixDev22/Awesome-Book-App
+
+### Run
+
+To run project, run the following command: git clone
+
+Open in desired editor.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Felix Nyamai
+
+- GitHub: [felixDev22](https://github.com/felixDev22)
+
+👤 Chimezie Asouzu
+
+- GitHub: [@Shegy28](https://github.com/shegy28)
+- Twitter: [@Shegy28](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/chimezie-asouzu-67704224a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Microverse learning experience
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give a star...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./mit.md) licensed.
+
```

Comment:
- [ ] You have a link of your license but this link is not working. 😨 kindly add a license file using these [guidelines ](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)